src/k8s-engine/minikube.js

```diff
@@ -313,8 +315,9 @@ class Minikube extends EventEmitter {
   async #installRancher() {
     // Ensure homestead is running
     console.log("starting homestead");
+    let mode = this.cfg?.rancherMode ?? "HOMESTEAD";
```

Comment:
Couldn't `??` be `||` because there is no falsey  value for `this.cfg.rancherMode` that we would want, right?

Reply:
Changed in ec3b065fe94aaaeafacfbf96aa5a43968bbc1ef4